src/main/java/use_cases/collaborative_task_management/collaborative_task_edit_use_case/CollaborativeTaskEditRequestModel.java

```diff
@@ -1,46 +1,44 @@
 package use_cases.collaborative_task_management.collaborative_task_edit_use_case;
 
-import entities.*;
+import entities.StudentUser;
```

Comment:
small issue: importing entities into Request model violates Common Closure & Single Responsibility Principle--see page 198 on the pdf of the CA book.

Reply:
Changed!